doc/en/example/pythoncollection.rst

```diff
@@ -313,3 +313,17 @@ interpreter:
     collect_ignore = ["setup.py"]
     if sys.version_info[0] > 2:
         collect_ignore_glob = ["*_py2.py"]
+
+User modules with names matching ``test*.py`` may be discovered as tests. Since 
+Pytest 2.6, users can prevent this test discovery by setting a boolean 
```

Comment:
```suggestion
Pytest 2.6, users can prevent pytest from discovering classes that start with ``Test`` by setting a boolean ``__test__`` attribute to ``False``.
```

Also I think we only need the example with `False` set; otherwise the example using `__test__ = True` implies that pytest will discover the class for tests even if it doesn't match the `python_classes` configuration option, which is not true.